feat: photoresister 드라이버 추가

min-kim-oss opened the pull request:
# ☝️Issue Number

- resolves #63

# 🔎 Key Changes

- photoresister_driver.h 추가
- photoresister_driver.c 추가
- bsw.c에  photoresister_driver.h 추가
- conf.oil 에   photoresister_driver.c

# 💌 To Reviewers

-

min-kim-oss committed:
feat: photoresister 드라이버 추가

fepick reviewed (approved):
asw.c는 UART 테스트를 위해 기존 파일로 두는 게 좋을 것 같습니다. 나머지는 괜찮음

fepick committed:
Feat/#64 uart_Driver 수신 기능 개선 (#68)

* feat: STM32 UART 통신 기본 헤더 및 소스코드 (#5)

* Update README.md

* Update README.md

* Initial commit

* feat: STM32보드 uart.h와 uart.c로 생성.
feat: UART_SendString, UART_SendFormatted 함수 생성
docs: README.md 생성 및 업데이트

@TODO: 함수명 컨벤션 확인 후 리팩토링 필요

---------

Co-authored-by: seheonnn <ho78901@naver.com>

* feat: 통신 패킷 설정

* feat: 통신 프로토콜 헤더 및 소스코드 작성 완료

* docs: 통신 프로토콜 정의 (#38)

* Update README.md

* Update README.md

* Initial commit

* cicd: GitHub Actions 스크립트 수정

* cicd: GitHub Actions 스크립트 수정

---------

Co-authored-by: Byeongmin Ryoo <103365736+bmryu0501@users.noreply.github.com>
Co-authored-by: Byeongmin Ryoo <bmryu0501@gmail.com>

* Delete STM32_main/Debug directory

실행파일이 있는 Debug 폴더를 삭제했습니다.

* docs: README 수정

* Revert "Docs/#46 README 수정"

* feat: uart_Driver 수신 기능 개선

---------

Co-authored-by: Byeongmin Ryoo <103365736+bmryu0501@users.noreply.github.com>
Co-authored-by: seheonnn <ho78901@naver.com>
Co-authored-by: Byeongmin Ryoo <bmryu0501@gmail.com>

seheonnn committed:
fix: fix conflict error
